locals.tf

```diff
@@ -46,4 +46,9 @@ locals {
     scope      = "terraform-managed"
     repository = "jenkins-infra/azure"
   }
+
+  trusted = {
```

Comment:
=> `trusted.ci.jenkins.io` 

- Avoid confusion with the "trusted" adjective
- full service name helps to map with the file `trusted.ci.jenkins.io.tf`